docs/matrix-creator/resources/microphone.md

```diff
@@ -0,0 +1,38 @@
+## Microphone Array on MATRIX Creator
+![Mic Position](../img/mic_creator_position.png)
+
+## Audio Specifications:
+
+Sample Rate: 8 to 48 kHz
+Bit Depth: 16 bit
+
+<!-- ## Position [x,y] of each mic in the array (mm): -->
+<h2 style="padding-top:0;">Position [x,y] of each mic in the array (mm):</h2>
+
+| Mic  |      X      |      Y      |  
+| ---- | ----------- | ----------- |  
+|  M1  |  20.0908795 | -48.5036755 |
+|  M2  | -20.0908795 | -48.5036755 |
+|  M3  | -48.5036755 | -20.0908795 |
+|  M4  | -48.5036755 |  20.0908795 |
+|  M5  | -20.0908795 |  48.5036755 |
+|  M6  |  20.0908795 |  48.5036755 |
+|  M7  |  48.5036755 |  20.0908795 |
+|  M8  |  48.5036755 | -20.0908795 |
+
+
+<h2 style="padding-top:0;">Connection to the FPGA:</h2>
+
+You can check the <a href="https://github.com/matrix-io/matrix-creator-fpga/blob/master/creator_core/creator.ucf" target="_blank">creator.ucf</a> file from the basic FPGA architecture. Here are the connections between the mics and the FPGA:
```

Comment:
It's not architecture. It's like: You can check the physical connection between the FPGA and other components in **creator.ucf**